src/main/webapp/data/userMapData.json

```diff
@@ -1321,8 +1342,10 @@
       "Glendale Community College",
       "Glenpool Public School",
       "Goldenwest College",
+      "Golf Platoon",
```

Comment:
This name doesn't tell us which military institute it's from - we might want to exclude it.